app/views/sections/index.html.erb

```diff
@@ -0,0 +1,100 @@
+<% require 'time' %>
+
+<div class = "menubody">
+
+    <h1>Food Menu for <%=Time.now.strftime("%A, %b %d")%></h1>
+    <div class = "center">
+
+      <h3 class = "toggle">~ CLICK TO HIDE MENU ITEMS ~</h3>
+      <h3 class = "toggle", style ="display: none">~ CLICK TO SHOW MENU ITEMS ~</h3>
+
+      <ul>
+        <% @sections.each do |section| %>
+        <li>
+          <a><h2><%= link_to section.name, section %></h2></a>
+        </li>
+          <%=section.description %>
+
+
+          <div class = "hide">
+           <ul>
+            <% @items.each do |item| %>
+              <% if item.menu_section_id == section.id %>
+                <li>
+                  <span class= "item"><%=item.name%></span>, <span class = "description"><%=item.description%></span> <span class = "price">(<%=item.price%>)</span> <%=item.note%>
+                </li>
+              <% end %>
+            <% end %>
+           </ul>
+          </div>
+          <div class = "buttons">
+            <li>
+              <form action="/sections/edit/<%=section.id %>" method="POST">
+                <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>" />
+                <input type="hidden" name="_method" value="GET" />
```

Comment:
For `_method` values that aren't `DELETE` or `PATCH` you can just put the attribute in the `<form>` element:

```html
<form action="/sections/edit/<%=section.id %>" method="GET">
```

This would allow you to remove line 34.